### PR TITLE
Merge redesign-welcoming-panel into main-development

### PR DESCRIPTION
**Pull request by lunaru100**
This branch also includes redesign of content browser panel and changes in ImguiLayer.cpp

### CODE REVIEW

File: Engine-Editor/src/EditorLayer.cpp [hunk 1]
```diff
@@ -297,48 +279,63 @@ namespace eg
...
-				{
-					if (ImGui::MenuItem("Open Project...", "Ctrl+O"))
-						OpenProject();
+			style.Colors[ImGuiCol_HeaderHovered] = ImVec4(0, 0, 0, 0);
+			style.Colors[ImGuiCol_HeaderActive] = ImVec4(0, 0, 0, 0);
+			style.Colors[ImGuiCol_TextDisabled] = m_LightTextShade;
 
-					ImGui::Separator();
+		//ImGui::Image((ImTextureID)m_LogoIcon->GetRendererID(),ImVec2(30,30));
+		//ImGui::SameLine();
```
Comment (miciex) [inline Engine-Editor/src/EditorLayer.cpp]: Delete these commented lines

File: Engine/src/Engine/Imgui/ImguiLayer.cpp [hunk 2]
```diff
@@ -87,67 +105,87 @@ namespace eg {
...
+			glfwMakeContextCurrent(backup_current_context);
+		}
+	}
+
+	void ImGuiLayer::SetDarkThemeColors()
+	{
+		/*m_LightTextShade = ImVec4(0.7765f, 0.7333f, 0.8863f, 1.0f);
+		m_LightShade = ImVec4(0.251f, 0.212f, 0.349f, 1.0f);
+		m_NormalShade = ImVec4(0.192f, 0.157f, 0.267f, 1.0f);
+		m_DarkShade = ImVec4(0.125f, 0.102f, 0.188f, 1.0f);
```
Comment (miciex) [inline Engine/src/Engine/Imgui/ImguiLayer.cpp]: Delete this comment

**Review by miciex — CHANGES_REQUESTED** [review]
Looks good to me besides the 2 places with unnecessary commented lines.

File: Engine/src/Engine/Imgui/ImguiLayer.cpp (hunk 2 above)
Comment (lunaru100) [reply Engine/src/Engine/Imgui/ImguiLayer.cpp]: this is actually useful, but I am gonna rework it anyway

**Review by miciex — APPROVED** [review]

**Pull request merged**